modules/redpanda/redpanda.go

```diff
@@ -252,6 +261,29 @@ func renderBootstrapConfig(settings options) ([]byte, error) {
 	return bootstrapConfig.Bytes(), nil
 }
 
+// validateListeners validates that the provided listeners are valid and set network aliases for the provided address.
+// The container must be attached to at least one network.
+func validateListeners(ctx context.Context, settings options, req testcontainers.GenericContainerRequest) error {
```

Comment:
Not a blocker:

I think this function it's correct, although for me it's doing two things in one: validate the listeners and define the network aliases.

Given the function name is `validate`, I could expect that the second part for setting the networks aliases is applied elsewhere.

I'm not sure how to improve this: a new name for the function (applyListeners??). PTAL at the comment in https://github.com/testcontainers/testcontainers-go/pull/1994/files#diff-ea8e8772fe279375ed6f3310607a32f3a18780f6ebfe9b3f817699b6afbdda4cR24-R35:

> // 4. Validate listeners and add network aliases to every network... 

If we rename the function, I'd change this too to

> // 4. Apply listeners to network aliases if they are valid... 

Wdyt?

Reply:
I agree with the new name

Reply:
I like it, thanks for understanding